src/Integration.Vsix.UnitTests/Commands/ProjectExcludePropertyToggleCommandTests.cs

```diff
@@ -37,24 +34,18 @@ public class ProjectExcludePropertyToggleCommandTests
         #region Test boilerplate
 
         private ConfigurableVsProjectSystemHelper projectSystem;
-        private IServiceProvider serviceProvider;
+
+        // TODO - cleanup. These unit tests are supposed to be testing
+        // ProjectExcludePropertyToggleCommand, so they don't need a real
+        // instance of ProjectPropertyManager. See #4770.
         private ProjectPropertyManager propertyManager;
         private Mock<IProjectToLanguageMapper> projectToLanguageMapper;
 
         [TestInitialize]
         public void TestInitialize()
         {
-            var provider = new ConfigurableServiceProvider();
-            this.projectSystem = new ConfigurableVsProjectSystemHelper(this.serviceProvider);
-            provider.RegisterService(typeof(IProjectSystemHelper), this.projectSystem);
-
-            var host = new ConfigurableHost(provider, Dispatcher.CurrentDispatcher);
-            propertyManager = new ProjectPropertyManager(host);
-            var mefExports = MefTestHelpers.CreateExport<IProjectPropertyManager>(propertyManager);
-            var mefModel = ConfigurableComponentModel.CreateWithExports(mefExports);
-            provider.RegisterService(typeof(SComponentModel), mefModel);
-
-            this.serviceProvider = provider;
```

Comment:
I'm not sure why some of these other types were being created - possibly not removed during a previous refactoring.

I considered refactoring the tests to remove the `[TestInitialize]`, but that would have introduced too much noise and obscured the real purpose of this PR.